Assets/SCORE.prefab

```diff
@@ -0,0 +1,299 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &1408636673887706897
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 3683644984303570021}
+  - component: {fileID: 4625418042019812531}
+  - component: {fileID: 632700510447983641}
+  m_Layer: 5
+  m_Name: SCORE
```

Comment:
Other prefabs follow `UpperCamelCase` name convention, not `UPPERCASE`. And let's store them in `Prefabs/` directory.

Reply:
Change to Score.prefab and moved to directory Prefabs/